pybtp/iutctl_common.py

```diff
@@ -149,6 +149,34 @@ def __init__(self):
         self.event_handler_cb = None
 
     def _rx_task(self):
+
```

Comment:
Please create a separate PR for this. This is Dialog-specific while other changes are independent.

Reply:
https://github.com/intel/auto-pts/pull/639